src/firefly/js/ui/Menu.css

```diff
@@ -67,6 +67,8 @@
     min-width: 800px;
     min-height: 400px;
     margin: -3px;
+    height: 100%;
+    width: 100%;
```

Comment:
Why were these added?  It should not if it were added to fix issues that's only related to router support.